.github/workflows/build.yml

```diff
@@ -167,7 +184,7 @@ jobs:
             build/pytest
 
   docs:
-    needs: [build, lint-js-test]
+    needs: [build]
```

Comment:
Looks like typedoc is built and uploaded in the lint job:

https://github.com/jupyterlite/jupyterlite/blob/989395371072ff32b7d407af5b05b37f77c504fd/.github/workflows/build.yml#L83-L89

Which makes the `docs` check flaky as these artifacts might not be available when the docs job runs.

Reply:
ah, good catch.